linkerd/manifest.json

```diff
@@ -2,7 +2,11 @@
   "maintainer": "help@datadoghq.com",
   "manifest_version": "1.0.0",
   "name": "linkerd",
+  "display_name": "Linkerd",
   "short_description": "Monitor your services health with metrics from linkerd.",
+  "metric_prefix": "rt.",
```

Comment:
The metric prefix should be `linkerd`. Seems like I did not prefixed the metric sin `metadata.csv`, sorry :/